install/check-minimum-requirements.sh

```diff
@@ -20,14 +20,14 @@ if [[ "$(vergte ${DOCKER_VERSION//v/} $MIN_DOCKER_VERSION)" -eq 1 ]]; then
 fi
 echo "Found Docker version $DOCKER_VERSION"
 
-COMPOSE_VERSION=$($dc_base version --short || echo '')
+COMPOSE_VERSION=$(docker compose version --short || echo '')
```

Comment:
`$dc_base` is removed, but I can keep it too if it's useful to keep that abstracted. The reason I removed it is because it seemed to only be used to determine if `docker compose` or `docker-compose` should be used

Reply:
+1 for removal for now.

Reply:
I bet we're going to hear about this from users. 🐭